Podfile.lock

```diff
@@ -1,20 +1,5 @@
 PODS:
   - 1PasswordExtension (1.8.5)
-  - AFNetworking (3.2.1):
```

Comment:
Any idea why the Podfile changes so much without Podfile changes?

Reply:
Yes, that's due to all the mucking about I did with making some of the pods local. I think I need to pull the original `Podfile` and `Podfile.lock` and overwrite the changes made in my PR.

Reply:
We've been working on nuking AFNetworking (with Sergio) during last Hack Week, from WordPressKit. This change is just (or should be!!) a side effect of running `pod install`.

No need to bring back the old Podfile.lock, this is a *beautiful* thing actually!!! (one less dependency).

Reply:
Ah, got it. 👍